src/components/app-header/app-header.tsx

```diff
@@ -1,53 +1,43 @@
+/* eslint-disable @stencil/own-methods-must-be-private */
+/* eslint-disable @stencil/strict-boolean-conditions */
```

Comment:
can we remove this ? or place it in the eslint file

Reply:
yes. I missed it